patches/hideFooter.css

```
@@ -0,0 +1,5 @@
+@media screen and (max-width: 1023px) {
+    footer {
+        display: none !important;
```

```suggestion
        display: none;
```
We're sticking to what we said in the comment